### PR TITLE
2 vital table

### LINKED ISSUE #2: Vitals table
Make Vital signs table

### PR DESCRIPTION
**Pull request by mingjiecn**
Vital table is done. closes #2

### CODE REVIEW

File: src/encoded/static/components/PatientChart.js [hunk 1]
```diff
@@ -0,0 +1,317 @@
...
+                this.viridisColors.push(allViridisColors[i]);
+            }
+        };
+        console.log(this.viridisColors);
+    }
+
+    componentDidMount() {
+        this.zingchart = window.zingchart;
+        this.moment = window.moment;
+        this.axios = window.axios;
```
Comment (cmonnom) [inline src/encoded/static/components/PatientChart.js]: We don't need axios anymore. Let's remove it.

File: src/encoded/static/components/patient.js [hunk 2]
```diff
@@ -27,6 +28,9 @@ class Patient extends React.Component {
         return (
             <div className={globals.itemClass(context, 'view-item')}>
                 <header className="row">
+                <script src="https://cdn.zingchart.com/zingchart.min.js"></script>
+                <script src="https://unpkg.com/axios@0.18.0/dist/axios.min.js" ></script>
```
Comment (cmonnom) [inline src/encoded/static/components/patient.js:32]: We don't need axios anymore. Let's remove it.

**Review by cmonnom — APPROVED** [review]
Looks good to me. Let's get rid of references to axios now that we don't need it anymore.

**Comment by mingjiecn** [comment]
Mingzhu's change is working on my local machine.

**Pull request merged**